src/plugins/layout/stateprovider.ts

```diff
@@ -129,3 +127,34 @@ export function createGenericLayoutState(node: Node, tag: string): ILayoutState
   };
   return layoutState;
 }
+
+export function shouldIgnoreNode(node: Node, shadowDom: ShadowDom): boolean {
+  let shadowNode = shadowDom.getShadowNode(getNodeIndex(node));
+  let ignore = false;
+  switch (node.nodeType) {
+    case Node.ELEMENT_NODE:
+      let tagName = (node as Element).tagName;
+      if (tagName === "SCRIPT" || tagName === "META") {
```

Comment:
Moved to a const like the other tags up top

---
In reply to: [132228127](https://github.com/Microsoft/clarity-js/pull/25#discussion_r132228127) [](ancestors = 132228127)